react/features/base/react/components/native/slidingviewstyles.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import { StyleSheet } from 'react-native';
+
+export const SIDEBAR_WIDTH = 250;
+
+export default {
+    /**
+     * The topmost container of the side bar.
+     */
+    sliderViewContainer: {
+        ...StyleSheet.absoluteFillObject,
+        zIndex: 1000
```

Comment:
do you think we should use a constants and keep track of these so we don't run into trouble? web does this, FWIW

Reply:
Yeah, probably!